pkg/network/networker.go

```diff
@@ -156,28 +157,21 @@ func (n *networker) attachYgg(id string, netNs ns.NetNS) (net.IPNet, error) {
 		return net.IPNet{}, fmt.Errorf("failed to generate ygg subnet IP: %w", err)
 	}
 
-	ips := []*net.IPNet{
-		&ip,
-	}
-
 	gw, err := n.ygg.Gateway()
 	if err != nil {
 		return net.IPNet{}, fmt.Errorf("failed to get ygg gateway IP: %w", err)
 	}
 
-	routes := []*netlink.Route{
-		{
-			Dst: &net.IPNet{
-				IP:   net.ParseIP("200::"),
-				Mask: net.CIDRMask(7, 128),
-			},
-			Gw: gw.IP,
-			// LinkIndex:... this is set by macvlan.Install
+	route := netlink.Route{
```

Comment:
I think even if this `now` has only a single route. It's important to understand that this won't always be the case. It's totally legal to configure multiple routes and taking away this ability is not recommended.